### PR TITLE
Modify parser to not recognize add and edit person command

### PR DESCRIPTION
**Pull request by ys112**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/NoteCommandTest.java [hunk 1]
```diff
@@ -35,19 +35,19 @@ void execute_addNoteUnfilteredList_success() {
...
+//
+//        NoteCommand executeCommand = new NoteCommand(INDEX_FIRST_PERSON, new Note("Updated Note"), true);
+//
+//        String expectedResult = String.format(executeCommand.MESSAGE_ADD_NOTE_SUCCESS, Messages.format(editedPerson));
+//
+//        Model expectedModel = new ModelManager(new AddressBook(model.getAddressBook()), new UserPrefs());
+//        expectedModel.setPerson(targetPerson, editedPerson);
+//
+//        assertCommandSuccess(executeCommand, model, expectedResult, expectedModel);
+//    }
```
Comment (lihaoquan) [inline src/test/java/seedu/address/logic/commands/NoteCommandTest.java:52]: Can check if this is commented out because it isn't in sync with Main?

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java [hunk 2]
```diff
@@ -64,29 +57,12 @@ public void parseCommand_delete() throws Exception {
...
     }
 
-    @Test
-    public void parseCommand_find() throws Exception {
-        List<String> keywords = Arrays.asList("foo", "bar", "baz");
-        FindCommand command = (FindCommand) parser.parseCommand(
-                FindCommand.COMMAND_WORD + " " + keywords.stream().collect(Collectors.joining(" ")));
-        assertEquals(new FindCommand(new NameContainsKeywordsPredicate(keywords)), command);
-    }
-
```
Comment (lihaoquan) [inline src/test/java/seedu/address/logic/parser/AddressBookParserTest.java:89]: Why is this removed?

**Review by lihaoquan — COMMENTED** [review]
Some questions for code removal / commented out codes.

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java (hunk 2 above)
Comment (ys112) [reply src/test/java/seedu/address/logic/parser/AddressBookParserTest.java:89]: Good catch. Left it out by mistake.

File: src/test/java/seedu/address/logic/commands/NoteCommandTest.java (hunk 1 above)
Comment (ys112) [reply src/test/java/seedu/address/logic/commands/NoteCommandTest.java:52]: Disabled to test, will enable it back again in new commit with the latest code.

File: src/test/java/seedu/address/logic/LogicManagerTest.java [hunk 3]
```diff
@@ -18,6 +19,7 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.io.TempDir;
 
+import seedu.address.logic.commands.AddApplicantCommand;
 import seedu.address.logic.commands.AddCommand;
```
Comment (lihaoquan) [inline src/test/java/seedu/address/logic/LogicManagerTest.java:23]: Potentially unused statement

**Review by lihaoquan — COMMENTED** [review]
LGTM other than 1 small issue.

File: src/test/java/seedu/address/logic/LogicManagerTest.java (hunk 3 above)
Comment (lihaoquan) [reply src/test/java/seedu/address/logic/LogicManagerTest.java:23]: Referring to `import seedu.address.logic.commands.AddCommand;`
